main/mySpace/myStudents.php

```diff
@@ -2057,8 +2057,26 @@ function ($hookHeader) {
     $form->display();
 }
 
+$coachAccessStartDate = null;
+$coachAccessEndDate = null;
+
+if (!empty($sessionId)) {
+    $filterMessages = api_get_configuration_value('filter_interactivity_messages');
+
+    if ($filterMessages) {
+        $sessionInfo = api_get_session_info($sessionId);
```

Comment:
Validate that $sessionInfo is not empty.

Reply:
$sessionId is validated on line 2063: https://github.com/chamilo/chamilo-lms/blob/ab409a8bf9c478d939798c12803e9a17924ed81d/main/mySpace/myStudents.php#L2063-L2067

Reply:
$sessionInfo can return an empty value

Reply:
Sorry, true, my mistake, I read on your comment $sessionId instead $sessionInfo.

Fixed: https://github.com/chamilo/chamilo-lms/blob/bbe79cfb3599afebcd2c6bf66bc1ed7883e6e159/main/mySpace/myStudents.php#L2068-L2071